gremlin-dotnet/src/Gremlin.Net/Process/Traversal/GraphTraversal.cs

```diff
@@ -203,6 +203,15 @@ private GraphTraversal(ICollection<ITraversalStrategy> traversalStrategies, Byte
             Bytecode.AddStep("as", args.ToArray());
             return Wrap<TStart, TEnd>(this);
         }
+        
+        /// <summary>
+        ///     Adds the asString step to this <see cref="GraphTraversal{SType, EType}" />.
+        /// </summary>
+        public GraphTraversal<TStart, TEnd> AsString<TNewEnd> ()
```

Comment:
Can't this return `GraphTraversal<TStart, string>`?

The types should usually be the same as in Java.